studio/pages/sign-in-fly-tos.tsx

```diff
@@ -0,0 +1,69 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import Image from 'next/image'
+import { Button } from 'ui'
+import { useTheme } from 'next-themes'
+import { API_URL } from 'lib/constants'
+
+const SignInFlyTos = () => {
+  const router = useRouter()
+  const {
+    query: { extension_id },
+  } = router
+  const { resolvedTheme } = useTheme()
+
+  const onSignInWithFly = async () => {
+    window.location.href = `${API_URL}/tos/fly?extension_id=${extension_id}`
+  }
+
+  return (
+    <div className="relative mx-auto flex flex-1 w-full flex-col items-center justify-center space-y-6">
+      <div className="absolute top-0 mx-auto w-full max-w-7xl px-8 pt-6 sm:px-6 lg:px-8">
+        <nav className="relative flex items-center justify-between sm:h-10">
+          <div className="flex flex-shrink-0 flex-grow items-center lg:flex-grow-0">
+            <div className="flex w-full items-center justify-between md:w-auto">
+              <Link href="/projects">
+                <a>
+                  <Image
+                    src={
+                      resolvedTheme === 'dark'
+                        ? `${router.basePath}/img/supabase-dark.svg`
+                        : `${router.basePath}/img/supabase-light.svg`
+                    }
+                    alt=""
+                    height={24}
+                    width={120}
+                  />
+                </a>
+              </Link>
+            </div>
+          </div>
+        </nav>
+      </div>
+      <div className="flex w-[320px] flex-col items-center justify-center space-y-3">
+        <h4 className="text-lg">Fly Single Sign On</h4>
```

Comment:
Not relevant to our users that this is SSO
```suggestion
        <h4 className="text-lg">Continue to Supabase Dashboard</h4>
```